apps/omg_eth/lib/eth/root_chain.ex

```diff
@@ -140,18 +141,69 @@ defmodule OMG.Eth.RootChain do
         input_txs_inclusion_proofs,
         in_flight_tx_sigs,
         from,
-        contract \\ nil
+        contract \\ nil,
+        opts \\ []
       ) do
-    opts =
-      @tx_defaults
-      |> Keyword.put(:value, @standard_exit_bond)
+    defaults = @tx_defaults |> Keyword.put(:value, @standard_exit_bond)
+    opts = defaults |> Keyword.merge(opts)
 
     contract = contract || from_hex(Application.fetch_env!(:omg_eth, :contract_addr))
     signature = "startInFlightExit(bytes,bytes,bytes,bytes)"
     args = [in_flight_tx, input_txs, input_txs_inclusion_proofs, in_flight_tx_sigs]
     Eth.contract_transact(from, contract, signature, args, opts)
   end
 
+  # credo:disable-for-next-line Credo.Check.Refactor.FunctionArity
+  def challenge_in_flight_exit_not_canonical(
+        inflight_txbytes,
+        inflight_input_index,
+        competing_txbytes,
+        competing_input_index,
+        competing_txid,
+        competing_proof,
+        competing_sig,
+        from,
+        contract \\ nil,
+        opts \\ []
+      ) do
+    defaults = @tx_defaults
+    opts = defaults |> Keyword.merge(opts)
+
+    contract = contract || from_hex(Application.fetch_env!(:omg_eth, :contract_addr))
+    signature = @challenge_in_flight_exit_signature
```

Comment:
consider renaming to sth like `func_signature` as we use signatures a lot and they mean something totally different